catkit/hardware/zwo/ZwoCamera.py

```diff
@@ -202,9 +259,14 @@ def just_take_exposures(self, exposure_time, num_exposures,
         :param gain: Gain of ZWO camera (volts).
         :param full_image: Boolean for whether to take a full image.
         :param bins: Integer value for number of bins.
+        :param use_video_capture_mode: Boolean for whether to use video capture or snapshot mode.
         :return: Two parameters: Image list (numpy data or paths), Metadata list of MetaDataEntry objects.
         """
 
+        # Set default for using video capture mode.
+        if use_video_capture_mode is None:
+            use_video_capture_mode = False
```

Comment:
[revision] Let's make the actual default ``False``, please, and let ``None := False``, which ``if use_video_capture_mode:`` does.

Reply:
I did this to let catkit, and only catkit, define the default, and still be able to pass through a use_video_capture from hicat-package.

You're saying set the default to False, but then hicat-package, would set the default. Or do you mean, manually covert Nones to Falses still, but just set the default to False? That doesn't sound clean to me.

Reply:
I mean:
```python
 def just_take_exposures(self, ... , use_video_capture_mode=False)
```

> You're saying set the default to False, but then hicat-package, would set the default.

I don't really follow what you mean here?

Reply:
Since you already define ``None := False``, they're logically the same thing, however, it's not obvious that this is actually the default as your dec is in the code and not the func sig.

Reply:
Okay, so you meant the latter.

But then you have multiple places in which the default is listed (1. in the function signature, and 2. in the function body). Setting the first one to True later on, and forgetting to change the second, will mean that the default from the just_take_exposures() call from HicatZwoCamera (in hicat-package, which currently has None as its default), will still mean False. By your logic, I would want to change the default in HicatZwoCamera as well. That default is now None, and should be False/True to make clear what the default is. The same is true for the simulator version of ZwoCamera (in hicat-package simulator). That would mean that you would need to keep at least *four* places all over the codebase in sync with each other to have a consistent default. And missing to change only one, means that you'll have inconsistent behaviour.

Setting all function signature defaults to None ensures that this will never happen. There is only one place where the default is defined (in the function body), and the docstring tells you what this default is (I'll add that after we agree on something, I forgot to do that before asking for your review).

Reply:
Changed the default parameters to False instead of None.